std-bits/base/src/main/java/org/enso/base/enso_cloud/logging/LogMessage.java

```diff
@@ -0,0 +1,8 @@
+package org.enso.base.enso_cloud.logging;
+
+public interface LogMessage {
+  /**
+   * @return JSON string representation of the log message.
+   */
+  String payload();
```

Comment:
Isn't this slightly over engineered? Why have a type for `LogMessage` at all, when each object has `toString()`?

Reply:
- I'd suggest to delete `LogMessage` and use `ch.qos.logback.classic.spi.ILoggingEvent` instead!
- then just extend [AppenderBase<ILoggingEvent>](https://github.com/enso-org/enso/pull/12163/files#diff-acf8b2c06bb249a5dd6e44720379976f375dd6e1d5d7abf87c03cd0bc2b0db26R26) 
- implement [append(ILoggingEvent ev) method](https://github.com/enso-org/enso/pull/12163/files#diff-acf8b2c06bb249a5dd6e44720379976f375dd6e1d5d7abf87c03cd0bc2b0db26R36)
- and we'll have the appender I want!

Reply:
> Isn't this slightly over engineered? Why have a type for `LogMessage` at all, when each object has `toString()`?

This is not a regular `toString`. Each log message in the API has some common JSON structure. IIRC the class encapsulates the common parts of that structure. Then the `payload` method converts the structure into a JSON string. But not any string is accepted and we still need this common logic for generating the JSON _somewhere_.

Reply:
Do we have any documentation describing the structure of all the messages?

>  Each log message in the API has some common JSON structure

That's the most important API. 

Whether the message content is obtained via `payload()` or `toString()` is a niché (but I still suggest to remove `LogMessage` in the long run).